dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -589,7 +589,13 @@ public void importScheme(final WorkflowSchemeImportExportObject workflowExportOb
             action = this.workflowAPI.findAction(permission.getInode(), user);
             if (null != action) {
 
-                this.permissionAPI.save(permission, action, APILocator.getUserAPI().getSystemUser(), false);
+                final Role role = roleAPI.loadRoleById(permission.getRoleId());
+                if (null != role) {
+                    this.permissionAPI.save(permission, action, APILocator.getUserAPI().getSystemUser(), false);
+                } else {
+                    Logger.warn(this, "On importing workflow, does not exists the permission role: "
```

Comment:
Include the Workflow Scheme ID in the error message too

Reply:
done